dali/test/python/test_fw_iterators_detection.py

```diff
@@ -110,30 +105,33 @@ def test_api_fw_check1_paddle():
 def test_api_fw_check1(iter_type, data_definition):
     root, annotations = data_paths()
     pipe = DetectionPipeline(BATCH_SIZE, 0, root, annotations)
-    train_loader = iter_type([pipe], data_definition, EPOCH_SIZE, auto_reset=False, dynamic_shape=True)
+    train_loader = iter_type([pipe], data_definition, EPOCH_SIZE, auto_reset=False,
+                             dynamic_shape=True)
     train_loader.__next__()
-    for method in [pipe.schedule_run, pipe.share_outputs, pipe.release_outputs, pipe.outputs, pipe.run]:
+    for method in [pipe.schedule_run, pipe.share_outputs, pipe.release_outputs, pipe.outputs,
+                   pipe.run]:
         try:
             method()
-            assert(False)
+            assert (False)
         except RuntimeError:
-            assert(True)
+            assert (True)
     # disable check
     pipe.enable_api_check(False)
-    for method in [pipe.schedule_run, pipe.share_outputs, pipe.release_outputs, pipe.outputs, pipe.run]:
+    for method in [pipe.schedule_run, pipe.share_outputs, pipe.release_outputs, pipe.outputs,
+                   pipe.run]:
         try:
             method()
-            assert(True)
+            assert (True)
         except RuntimeError:
-            assert(False)
+            assert (False)
```

Comment:
Why do we even need/want this check? Can't we just let the error be thrown and _see_ it in case of failure?

Reply:
I guess it helps with figuring out, what this test is for. Anyway, I wouldn't like to modify test behaviour significantly and I regard this as being the "too much" change in the formatting-adjustment PR. I removed the parentheses from the `assert` to treat this as a keyword. But I'd like to keep the try-catch, even if it is redundant.